### DISCUSSION TITLE
Thanks to all

### DISCUSSION THREAD

**Discussion by BarbaPeru** [zellij-org/zellij#1302] [General]
Hallo, a message to thanks all the developers for this great piece of software, great work. I compiled it from scratch and I'm using it since a couple of days and I'm already in love. I like the power of the tool, the default keybindings, but my muscle memory thanks for tmux keybindings emulation too ;-)

I would like a more modal approach (like vim) for composing commands, such as:

enter command mode
t3l (means move three tabs to the left)

But you are doing a good job, so continue this path.

Thanks.

**Comment by jaeheonji** [comment]
Thank you @BarbaPeru 😄 
I am also happy to hear that you are enjoying zellij.

The feature you suggested looks very good. but, we will have to see if that is possible for all actions. 
Anyway, overall, I agree with your suggestion 👍
